packages/puppeteer-extra-plugin-block-resources/index.js

```diff
@@ -111,6 +112,6 @@ class Plugin extends PuppeteerExtraPlugin {
   }
 }
 
-module.exports = function(pluginConfig) {
+module.exports = function (pluginConfig) {
```

Comment:
Can you revert this change please?

There's no strict guidelines on how the code should be linted, therefore it's best to keep it as similar as possible for now :)